olps/datasources/ac_datasource.py

```diff
@@ -0,0 +1,32 @@
+from typing import *
+from math import log10
+from .datasource import DataSource
+import numpy as np
+
+
+class ACDataSource(DataSource):
+    # A matrix of price relative vectors.
+    # Each column is a logarithmic PRV, and the time increases from left to right.
+    log_price_relatives: np.array
+    last_log_price_relatives: np.array
+
+    def __init__(self, initial_prices: np.array):
+        """
+        Initialize the data source with the price of all assets at the beginning of strategy execution.
+        """
+        DataSource.__init__(self, initial_prices=initial_prices)
```

Comment:
Avoid mentioning the superclass by name when working with inheritance in Python. It's not wrong, it's just not necessarily idiomatic, and is bad practice in case of multiple inheritance.

See [this SO post for why](https://stackoverflow.com/a/576183/6412600).